lib/cppcheck.cpp

```diff
@@ -895,7 +895,7 @@ unsigned int CppCheck::checkInternal(const FileWithDetails& file, const std::str
     if (Settings::terminated())
         return mLogger->exitcode();
 
-    const Timer fileTotalTimer(mSettings.showtime == SHOWTIME_MODES::SHOWTIME_FILE_TOTAL, file.spath());
+    Timer WholeFileTimer{file.spath(), mSettings.showtime};
```

Comment:
variable name should not start with capital. and since this is the timer you activate with "--showtime=file-total" I like "fileTotal" in the name?